i18n/sk.json

```diff
@@ -156,7 +156,7 @@
     "Shorten URL": "Skrátiť URL",
     "Editor": "Editor",
     "Preview": "Náhľad",
-    "%s requires the PATH to end in a \"%s\". Please update the PATH in your index.php.": "%s vyžaduje, aby PATH končila na \"%s\". Aktualizujte prosím PATH vo svojom index.php.",
+    "%s requires the PATH to end in a \"%s\". Please update the PATH in your index.php.": "%s vyžaduje, aby PATH končila na „%s\". Aktualizujte prosím PATH vo svojom index.php.",
```

Comment:
Incorrect escaping

Reply:
Apparently Slovak uses those as starting quotation marks, see for example https://op.europa.eu/en/web/eu-vocabularies/formex/physical-specifications/character-encoding/use-of-quotation-marks-in-the-different-languages - but since we don't apply quotation mark rules to other translations or other places in this one, I've reverted it, for consistency.